x/superfluid/keeper/hooks_test.go

```diff
@@ -283,6 +303,13 @@ func (suite *KeeperTestSuite) TestBeforeSlashingUnbondingDelegationHook() {
 				suite.Require().NoError(err)
 				// slash by slash factor
 				power := sdk.TokensToConsensusPower(validator.Tokens, sdk.DefaultPowerReduction)
+
+				// should not be slashing unbonded validator
```

Comment:
asking to learn : does it not matter for the unbonding case?

Reply:
Yes i tested it does not matter for the unbonding case?